tests/storage/utils.py

```
@@ -527,3 +528,75 @@ def check_file_in_vm(
+        scheduled_statuses: set[str] = {
+            DataVolume.Status.IMPORT_SCHEDULED,
+            DataVolume.Status.ClONE_SCHEDULED,
+            DataVolume.Status.UPLOAD_SCHEDULED,
+        }
+        if phase in scheduled_statuses:
+            now = time.monotonic()
+            if phase != last_value:
+                last_value = phase
+                last_change_time = now
+            elif now - last_change_time > 300:
+                LOGGER.error(f"DV {dv.name} stuck in {phase} for 5 minutes")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)